packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -49,6 +49,11 @@ abstract class ScrollActivityDelegate {
   /// Terminate the current activity and start a ballistic activity with the
   /// given velocity.
   void goBallistic(double velocity);
+
+  /// Update the ballistic animation instead of restarting  it, for example as the result of a layout change after a flinging gesture.
+  ///
+  /// The [initVelocity] and [initPosition] refer to the starting values of the new ballistic animation.
+  Simulation? updateBallisticAnimation(double initVelocity, double initPosition);
```

Comment:
document: when does this return null? What does that mean?

Reply:
added